provider-ui/ui_dev_server.js

```diff
@@ -23,7 +20,10 @@ proxy.on('error', (err, req, res) => {
 
 app.prepare().then(() => {
   createServer((req, res) => {
-    const { pathname } = parse(req.url, true);
+    // Be sure to pass `true` as the second argument to `url.parse`.
+    // This tells it to parse the query portion of the URL.
+    const parsedUrl = parse(req.url, true)
```

Comment:
I guess we can revert these changes (maybe keep comment)? Since we are only using pathName that was already desctructured from parse function...

Reply:
👍